src/App.tsx

```diff
@@ -91,27 +100,69 @@ const App: React.FC<any> = () => {
     });
   }, [apiEndpoint]);
 
-  return <Suspense fallback={<Loading />}>
-    <EndpointModal visible={isEndpointModalVisible} setModalState={setEndpointModalState} />
-    <Routes>
-      <Route path="tests/*" element={<Tests />} />
-      <Route path="test-suites/*" element={<TestSuites />} />
-      <Route path="executors/*" element={<Executors />} />
-      <Route path="sources/*" element={<Sources />} />
-      <Route path="triggers" element={<Triggers />} />
-      <Route path="settings" element={<GlobalSettings />} />
-      <Route
-        path="/apiEndpoint"
-        element={isApiEndpointLocked() ? <Navigate to="/" replace /> : <EndpointProcessing />}
-      />
-      <Route path="/" element={<Navigate to="/tests" replace />} />
-      <Route path="*" element={<NotFound />} />
-    </Routes>
-    {isFullScreenLogOutput ? <LogOutputHeader logOutput={logOutput} isFullScreen /> : null}
-    <CSSTransition nodeRef={logRef} in={isFullScreenLogOutput} timeout={1000} classNames="full-screen-log-output" unmountOnExit>
-      <FullScreenLogOutput ref={logRef} logOutput={logOutput} />
-    </CSSTransition>
-  </Suspense>;
+  return (
+    <Suspense fallback={<Loading />}>
+      {!isTestkubeCloudLaunchBannerHidden && (
```

Comment:
It's better to use `condition ? <Component /> : null` instead of `condition && <Component />`, to avoid printing `condition` in case it's falsy.

Reply:
I'm thinking about this all the time. Let's use `.. ? .. : ..`, agreed ✅